resources/openebs-operator-1.3.0.yaml

```diff
@@ -0,0 +1,751 @@
+        # of cstor pool pod. This ENV is also used to indicate the location
+        # of the sparse devices.
+        # The default path used is /var/openebs/sparse
+        - name: OPENEBS_IO_CSTOR_POOL_SPARSE_DIR
+          value: "/var/lib/gravity/openebs/sparse"
+        # OPENEBS_IO_JIVA_POOL_DIR can be used to specify the hostpath
+        # to be used for default Jiva StoragePool loaded by OpenEBS
+        # The default path used is /var/openebs
+        # This value takes effect only if OPENEBS_IO_CREATE_DEFAULT_STORAGE_CONFIG
+        # is set to true
+        - name: OPENEBS_IO_JIVA_POOL_DIR
+          value: "/var/lib/gravity/openebs"
+        # OPENEBS_IO_LOCALPV_HOSTPATH_DIR can be used to specify the hostpath
+        # to be used for default openebs-hostpath storageclass loaded by OpenEBS
+        # The default path used is /var/openebs/local
+        # This value takes effect only if OPENEBS_IO_CREATE_DEFAULT_STORAGE_CONFIG
+        # is set to true
+        - name: OPENEBS_IO_LOCALPV_HOSTPATH_DIR
+          value: "/var/lib/gravity/openebs/local"
+        - name: OPENEBS_IO_JIVA_CONTROLLER_IMAGE
+          value: "registry.local:5000/openebs/jiva:1.3.0"
+        - name: OPENEBS_IO_JIVA_REPLICA_IMAGE
+          value: "registry.local:5000/openebs/jiva:1.3.0"
+        - name: OPENEBS_IO_JIVA_REPLICA_COUNT
+          value: "3"
+        - name: OPENEBS_IO_CSTOR_TARGET_IMAGE
+          value: "registry.local:5000/openebs/cstor-istgt:1.3.0"
+        - name: OPENEBS_IO_CSTOR_POOL_IMAGE
+          value: "registry.local:5000/openebs/cstor-pool:1.3.0"
+        - name: OPENEBS_IO_CSTOR_POOL_MGMT_IMAGE
+          value: "registry.local:5000/openebs/cstor-pool-mgmt:1.3.0"
+        - name: OPENEBS_IO_CSTOR_VOLUME_MGMT_IMAGE
+          value: "registry.local:5000/openebs/cstor-volume-mgmt:1.3.0"
+        - name: OPENEBS_IO_VOLUME_MONITOR_IMAGE
+          value: "registry.local:5000/openebs/m-exporter:1.3.0"
+        - name: OPENEBS_IO_CSTOR_POOL_EXPORTER_IMAGE
+          value: "registry.local:5000/openebs/m-exporter:1.3.0"
+        # OPENEBS_IO_ENABLE_ANALYTICS if set to true sends anonymous usage
+        # events to Google Analytics
+        - name: OPENEBS_IO_ENABLE_ANALYTICS
+          value: "false"
+        - name: OPENEBS_IO_INSTALLER_TYPE
+          value: "openebs-operator"
+        # OPENEBS_IO_ANALYTICS_PING_INTERVAL can be used to specify the duration (in hours)
+        # for periodic ping events sent to Google Analytics.
+        # Default is 24h.
+        # Minimum is 1h. You can convert this to weekly by setting 168h
+        #- name: OPENEBS_IO_ANALYTICS_PING_INTERVAL
+        #  value: "24h"
+        livenessProbe:
+          exec:
+            command:
+            - /usr/local/bin/mayactl
+            - version
+          initialDelaySeconds: 30
+          periodSeconds: 60
+        readinessProbe:
+          exec:
+            command:
+            - /usr/local/bin/mayactl
+            - version
```

Comment:
I'm sure this is just pulled from some reference, seems odd though that readiness/liveness probes would be based on the ability to run version on the cli?

Reply:
Yeah, this is just their spec, like mentioned above - I didn't write it myself.

Reply:
Looking at their docs I think this command tries to contact apiserver as well, that's why they use it in the probes: https://docs.openebs.io/docs/next/mayactl.html#using-mayactl (at the bottom).